src/macros.rs

```diff
@@ -22,8 +28,8 @@ macro_rules! itry {
     })
 }
 
-/// Unwrap the given Option or return a Ok(Response::new()) with the given modifier. The default
-/// modifier is status::BadRequest.
+/// Unwrap the given `Option` or return a `Ok(Response::new())` with the given
+/// modifier. The default modifier is `status::BadRequest.
```

Comment:
Jeez, sorry for merging :(

On 10 March 2016 11:58:14 CET, Skyler <notifications@github.com> wrote:
>> @@ -22,8 +28,8 @@ macro_rules! itry {
>>      })
>>  }
>>  
>> -/// Unwrap the given Option or return a Ok(Response::new()) with the
>given modifier. The default
>> -/// modifier is status::BadRequest.
>> +/// Unwrap the given `Option` or return a `Ok(Response::new())` with
>the given
>> +/// modifier. The default modifier is `status::BadRequest.
>
>Missing a backtick here
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/iron/iron/pull/435/files#r55663328

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.